README.md

```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that these flags are set for you.
+
+Currently in `Build, Execution, Deployment -> Build Tools -> Gradle -> Runner` (but just search for `runner`)
+
+ * set "Delegate IDE build/run actions to gradle" to true
+ * set "Run test using:" to "Gradle Test Runner"
+
+Alternatively, you can:
+
+ * copy your quasar.jar to the `<project root>/lib/` dir
```

Comment:
Which quasar.jar is "your" quasar.jar? Do they have to download it online? How do they pick a version?

Reply:
Updated a bit - do you think it's enough?